apiml-security-common/src/main/java/org/zowe/apiml/security/common/auth/saf/PlatformErrno2.java

```diff
@@ -0,0 +1,1812 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package org.zowe.apiml.security.common.auth.saf;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import lombok.Generated;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Provides explanation for secondary error codes for as described at:
+ * https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.4.0/com.ibm.zos.v2r4.bpxa800/errnojrs.htm
+ */
```

Comment:
I think there is missing "l' at the end of the link, https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.4.0/com.ibm.zos.v2r4.bpxa800/errnojrs.html